MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -175,9 +185,14 @@ public void UpdateEncoderPath(string path, string pathType)
                 newPath = path;
             }
 
+            // Don't save an invalid path
+            if (!File.Exists(path))
```

Comment:
This doesn't work, right?
I can't enter "ffmpeg" in the web interface bc it will 404

Reply:
Should you be able to?

Reply:
IMO, yes

Reply:
Removed